src/components/Menus/Sidebar/InstructionsPanel/InstructionsPanel.jsx

```diff
@@ -79,13 +81,30 @@ const InstructionsPanel = () => {
     }
   }, [quizCompleted, currentStepPosition, numberOfSteps, dispatch, isQuiz]);
 
+  useEffect(() => {
+    dispatch(setProjectInstructions(editInstructions));
+  }, [editInstructions, dispatch]);
+
+  useEffect(() => {
+    if (project?.instructions) {
+      setEditInstructions(project.instructions);
+    }
+  }, [project, dispatch]);
+
```

Comment:
I was a bit confused by this... Is there a reason we need to use `editInstructions` instead of reading the value straight out of the store? And similarly for `setEditInstructions` - why can't we just call the dispatch directly when the text in the textarea changes?

Reply:
Good point, it wasn't needed. I've removed the useState approach in [98423e6](https://github.com/RaspberryPiFoundation/editor-ui/pull/1163/commits/98423e62e0f060987444977302e01ed0a4e2de0e)